app/controllers/application_controller.rb

```diff
@@ -50,4 +53,49 @@ def redirect_if_logged_in(path = "/dashboard")
 
     redirect_to path, notice: I18n.t("already_logged_in_notice")
   end
+
+  def generate_user_jwt(user)
+    payload = {
+      email: user.email,
+      sub: user.token,
+      exp: 24.hours.from_now.to_i
+    }
+
+    JWT.encode(payload, ENV.fetch('JWT_SECRET', nil), 'HS256')
+  end
+
+  def send_user_jwt_to_phoenix(jwt)
+    uri = URI("#{ENV.fetch('PHOENIX_URI', nil)}/api/external_login")
```

Comment:
maybe it should early return if there is no PHOENIX_URI configured? that way we can still run standalone on local dev envs and it won't fail or even try to hit the phoenix app unless it's configured?
```suggestion
    return if Rails.configuration.phx_interop.phx_uri.blank?
    uri = URI("#{Rails.configuration.phx_interop.phx_uri}/api/external_login")
```